action.yml

```diff
@@ -53,20 +53,44 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Check if review is requested
+      shell: bash
+      run: |
+        if [[ "${{ github.event.pull_request.title }}" =~ ^(no|skip)(-|\s)?review|skip(-|\s)?code(-|\s)?review|otter(ai)?(-|\s)?skip|otter(-|\s)?restricted ]]; then
```

Comment:
🔍 Ensure the regex pattern covers all necessary cases for skipping reviews. Review the conditions to avoid false negatives.